src/atoms/Value.jsx

```diff
@@ -3,7 +3,7 @@ import Color from "../data/Color";
 
 import RotateContext from "../context/RotateContext";
 
-const Value = ({ value, fixed }) => {
+const Value = ({ value, fixed, outerBorderColor="" }) => {
```

Comment:
This can just be `outerBorderColor` which would default to `undefined` which will work in your if below.
```suggestion
const Value = ({ value, fixed, outerBorderColor }) => {
```